tokio/src/runtime/task/join.rs

```diff
@@ -162,7 +162,8 @@ impl<T> JoinHandle<T> {
     ///
     /// Awaiting a cancelled task might complete as usual if the task was
     /// already completed at the time it was cancelled, but most likely it
-    /// will complete with a `Err(JoinError::Cancelled)`.
+    /// will complete with an `Err(JoinError)` and can be checked that
+    /// it was cancelled via [`JoinError::is_cancelled`].
```

Comment:
I think this made it harder to read. How about this?
```suggestion
    /// will fail with a cancelled `JoinError`.
```

Reply:
maybe the word `cancelled` could link to the `JoinError::is_cancelled` docs without being the _name_ of the function?
```suggestion
    /// will fail with a [cancelled] `JoinError`.
    ///
    /// [cancelled]: JoinError::is_cancelled
```

Reply:
@Darksonn @hawkw Thank you for suggestions! The `cancelled` linked to the method looks clear. I've pushed an update.